tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -176,12 +176,13 @@ def _init_devices(self):
                 return
 
             for device in settings.GOLDEN_DEVICES:
+                port, _ = device
```

Comment:
I think the code here should be fine.  
Line 171 has already filtered out non-OpenThread devices by `settings.GOLDEN_DEVICE_TYPE`